R/convGDX2MIF.R

```diff
@@ -126,6 +126,30 @@ convGDX2MIF <- function(gdx, gdx_ref = NULL, file = NULL, scenario = "default",
     message("function reportSDPVariables does not work and is skipped")
   }
 
+  # Report climate assessment variables & merge with output
+  message("running reportClimate...")
+  climateVars <- reportClimate(gdx) # Returns NULL if no climate assessment variables are available
+  if (!is.null(climateVars)) {
```

Comment:
All this should be moved to `reportClimate`. convGDX2MIF is the hub function and should only have two lines of code, sth along the lines of

```
message("running reportClimate...")
output <- mbind(output,reportClimate(gdx, ...))
```

Reply:
Moved to `reportClimate.R`